gopher.go

```diff
@@ -257,6 +258,7 @@ func main() {
 					`- <https://segment.com/blog/allocation-efficiency-in-high-performance-go-services/>`,
 				}, "\n"),
 			),
+			handlers.RespondTo([]string{"threads"}, "Out of respect for the visually impaired, please refrain from using Slack threads, as they are not accessible for screen readers. Instead, post your replies within the channel."),
```

Comment:
This probably needs my copy suggestion from below.

Reply:
Heard. Updates coming soon.